x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/infra_logs/kpi_over_time_config.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ConfigProps, SeriesConfig } from '../../types';
+import { FieldLabels, RECORDS_FIELD, ReportTypes } from '../constants';
+import { LOG_RATE as LOG_RATE_FIELD } from '../constants/field_names/infra_logs';
+import { LOG_RATE as LOG_RATE_LABEL } from '../constants/labels';
+
+export function getLogsKPIConfig(configProps: ConfigProps): SeriesConfig {
+  return {
+    reportType: ReportTypes.KPI,
+    defaultSeriesType: 'bar',
+    seriesTypes: [],
+    xAxisColumn: {
+      label: i18n.translate('xpack.observability.exploratoryView.logs.logRateXAxisLabel', {
+        defaultMessage: 'Timestamp',
+      }),
+      dataType: 'date',
+      operationType: 'date_histogram',
+      sourceField: '@timestamp',
+      isBucketed: true,
+      scale: 'interval',
+    },
+    yAxisColumns: [
+      {
+        label: i18n.translate('xpack.observability.exploratoryView.logs.logRateYAxisLabel', {
+          defaultMessage: 'Log rate per minute',
+        }),
+        dataType: 'number',
+        operationType: 'count',
+        isBucketed: false,
+        scale: 'ratio',
+        sourceField: RECORDS_FIELD,
+        timeScale: 'm',
+      },
+    ],
+    hasOperationType: false,
+    filterFields: ['agent.type', 'service.type', 'event.dataset'],
+    breakdownFields: ['agent.hostname', 'service.type', 'event.dataset'],
+    baseFilters: [],
+    definitionFields: ['agent.hostname', 'service.type', 'event.dataset'],
```

Comment:
I think the difference essentially comes down to the ability to negate on one. And the other allows supplying a filter to filter down the available values. We could potentially just use one, maybe 🤔 

> For some reason the event.dataset field seems to have a virtual ALL_VALUES option which overshadows a real event.dataset of that value. Is that inherent to the exploratory view or something we can control?

it’s not something I’ve added. I can take a look to see if we can control it somehow. I hadn’t really considered someone having a real ALL_VALUES value.

Reply:
> Just leaving the fields empty should work, shouldn't it?

I assume it should. Like I mentioned to Kate I'm not 100% sure how the validation there works (haven't had a chance to check yet).

> Maybe it would work if we moved all of the fields to be filters instead?

Yeah, that should work for our use case since they're literally just duplicates of each other. So `message` would remain as a definition field, and everything else would be a filter? @jasonrhodes do you have any thoughts on this one?

Reply:
That sounds fine to me. I'm not too concerned about this experience being absolutely perfect at first, so whatever gets it out there without being broken seems ok to me. Thanks for digging in here and validating what works and what doesn't!

Reply:
I'm still not getting the difference between definition field and filter, sorry. Wouldn't the `message` field make for a good filter as well?

Reply:
Technically there is no difference. Think of definition field as something, around which usually UI is build. 
For apm it would be Service name or Environement filter, for uptime it's monitor name or monitor id. 
For metrics, i guess it could be host name.